shopfloor/migrations/16.0.2.0.0/post-migration.py

```diff
@@ -37,7 +37,5 @@ def _enable_option_in_menus(menus):
     for menu in menus:
         menu.allow_alternative_destination_package = True
         _logger.info(
-            "Option allow_alternative_destination_package enabled for menu {}".format(
-                menu.name
-            )
+            f"Option allow_alternative_destination_package enabled for menu {menu.name}"
```

Comment:
As it's a logger, string interpolation should be done through it:
```suggestion
            Option allow_alternative_destination_package enabled for menu {}",
            menu.name,
```

Reply:
This is a logger of a migration step...  TBH I don't mind that much. And it will be trashed in 18.